msteams-platform/toolkit/configure-message-extension-capability.md

```diff
@@ -0,0 +1,632 @@
+---
+title: Configure-message-extension-capability
+author: surbhigupta
+description: Learn to configure message extension capability within your Teams app.
+ms.author: v-vanv
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 03/01/2024
+---
+
+# Configure Message extension capability within your Teams app
+
+Message extension allows users to interact with your web service when composing messages in Microsoft Teams. Users can invoke your web service to assist message composition from the message compose box or from the search bar.
+
+Message extensions are implemented on top of the bot support architecture within Teams. For more information, see [Build message extensions](../messaging-extensions/what-are-messaging-extensions.md).
+
+## Prerequisites
+
+To configure message extension as an additional capability, ensure the following prerequisites:
+
+* App manifest (previously called Teams app manifest).
+* [Microsoft 365 account](../concepts/build-and-test/prepare-your-o365-tenant.md) to test the application.
+* Microsoft Azure Storage account.
+
+To add message extension to a Teams tab app, see [Add message extension to Teams tab app](#add-message-extension-to-teams-tab-app).
+
+To add message extension to a Teams bot app, see [Add message extension to Teams bot app](#add-message-extension-to-teams-bot-app).
+
+## Add message extension to Teams tab app
+
+The following steps help you to add message extension capability to a tab app:
+
+1. [Create message extension app using Microsoft Teams Toolkit](#create-message-extension-app-using-microsoft-teams-toolkit)
+1. [Configure Message extension capability within your Teams app](#configure-message-extension-capability-within-your-teams-app)
+1. [Bring message extension code to your project](#bring-message-extension-code-to-your-project)
```

Comment:
Add message extension code to your project

Reply:
done